### DISCUSSION TITLE
PictureChooser deleted from plugin in Add .NET 6.0 Support commit (#4319)

### DISCUSSION THREAD

**Discussion by jbouche-work** [MvvmCross/MvvmCross#4444] [Q&A] [answered]
Hi, I've seen that in the commit [f76f09f] 29/06, the PictureChooser is removed from the plugins with comment : "* Remove useless plugins".
It was not compatible with .Net 6 ?
Will it be re-added in the next PR ?
Another plugin is recommended ?

My need is to reuse it for add multi-selection of picture adding a method like "ChooseMultiplePicturesFromLibraryAsync".

**Comment by Cheesebaron** [comment, marked as answer]
Just use Xamarin.Essentials

**Reply by jbouche-work** [reply]
Hi,
If I use Xamarin.Essentials, I lose maxPixelDimension (resize) and percentQuality (compress) feature ?

**Reply by Cheesebaron** [reply]
Too bad. You have to resize images yourself.
